test/e2e/playwright.config.ts

```diff
@@ -0,0 +1,37 @@
+import type { PlaywrightTestConfig } from '@playwright/test'
+
+const config: PlaywrightTestConfig = {
+  testDir: '.',
+  webServer: {
+    /**
+     * Note: You can speed up local testing by switching `prod` out for `start`
```

Comment:
It actually occurred to me sometime last night that we could add like a `slow-start` script that sleeps for a second or two before actually starting the nuxt server.

Or we could combine it with the [`wait-for` script that I was using](https://github.com/Eficode/wait-for) before I discovered that Playwright had this `webserver` option built into it. `start` could see if `API_URL` was pointing to localhost and then `wait-for $API_URL -- pnpx nuxt start` or something.